<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowFileDlg</class>
 <widget class="QDialog" name="ShowFileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="shLblTitle">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="shLblDatePlace">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shBtnSeeImage">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查看相册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shBtnEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shBtnDelete">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shBtnReturn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="shTetText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>561</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>shBtnReturn</sender>
   <signal>clicked()</signal>
   <receiver>ShowFileDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
